xayn-ai/src/coi/relevance.rs

```diff
@@ -144,6 +145,7 @@ impl RelevanceMap {
         self.coi_to_relevance
             .remove(&coi_id)
             .map(|relevances| {
+                self.cleaned.remove(&coi_id);
```

Comment:
is there a specific reason that this call is inside the closure? otherwise it could be called outside.